src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -94,7 +97,9 @@ fun collectPathResolveVariants(
         if (e.name == referenceName) {
             val element = e.element ?: return@createProcessor false
             if (element !is RsDocAndAttributeOwner || element.isEnabledByCfgSelf) {
-                result += BoundElement(element, e.subst)
+                val boundElement = BoundElement(element, e.subst)
+                val isPublic = e.isVisibleFrom(containingMod)
+                result += BoundElementWithVisibility(boundElement, isPublic)
```

Comment:
Let's rename `isPublic` to `isVisible` in the remaining places like this variable and BoundElementWithVisibility.isPublic and ResolvedPath.Item.isPublic